pkg/etcdcli/cached_members.go

```diff
@@ -117,3 +117,7 @@ func (c *CachedMembers) UnhealthyVotingMembers(ctx context.Context) ([]*etcdserv
 
 	return filterVotingMembers(c.members.GetUnhealthyMembers()), nil
 }
+
+func (c *CachedMembers) MemberHealth(ctx context.Context) (memberHealth, error) {
```

Comment:
i just returned the `memberhealth` as it is always updated using the mutex iiuc

Reply:
I would still return a defensive copy here. Don't we have to export the "memberHealth" struct?

Reply:
on it 👍🏽

Reply:
exporting `memberHealth` will cause a lot of changes all over the package, also it duplicates with 

```
type MemberHealth interface {
	MemberHealth(ctx context.Context) (memberHealth, error)
}
```

i think using exporting the method will do 👍🏽

Reply:
yep, sounds good. I think it's a typedef for the slice anyway, so we're fine.